src/main/java/com/bp3x/raidbot/commands/lfg/DeleteCommand.java

```diff
@@ -57,7 +57,6 @@ protected void execute(CommandEvent commandEvent) {
            MessageUtils.sendAutoDeletedMessage("Invalid arguments. Use `!delete <eventID>`", 300, commandEvent);
         }
         if (eventToDelete != null && messageToDelete != null) {
-            MessageUtils.autoDeleteMessage(commandEvent.getMessage(), 300);
```

Comment:
This line was unnecessary because sendAutoDeletedMessage removes the commandEvent message